ludwig/features/sequence_feature.py

```diff
@@ -301,8 +302,9 @@ def get_prediction_set(self):
     def get_output_dtype(cls):
         return tf.int32
 
-    def get_output_shape(self):
-        return self.max_sequence_length,
+    @property
+    def output_shape(self) -> torch.Size:
+        return torch.Size((self.max_sequence_length, ))
```

Comment:
As above, we should stick to either list or tuple.

Reply:
Done